page-templates/flexible.php

```diff
@@ -1,11 +1,17 @@
 <?php /* Template Name: Flexible */
 
-use \Timber\Timber;
-
-get_header();
+use Timber\Timber;
+use LeanNs\Assets;
 
 $data = Timber::get_context();
 $data['post'] = \LeanNs\Patterns::get_post();
+
+if ( 'images' === $data['post']->acf['hero']['hero_background']['type'] ) {
```

Comment:
Yes I think an exception makes sense in this scenario, however we need to add that into the code, since this is going to trigger an error based on the type of configuration of the `env`